cli/src/pcluster/cli/commands/configure/utils.py

```diff
@@ -203,6 +205,32 @@ def get_regions():
     return regions
 
 
+@handle_client_exception
+def instance_type_supports_efa(instance_type: str) -> bool:
+    """Return whether of not a specified instance type supports EFA.
+
+    It uses the instance type information returned from AWS API.
+    :param instance_type:
+    :return: True if instance type support EFA, False otherwise
+    """
+    return AWSApi.instance().ec2.get_instance_type_info(instance_type).is_efa_supported()
+
+
+@handle_client_exception
+def placement_group_exists(placement_group_name: str) -> bool:
+    """Return whether of not a specified placement group exists based on the placement group name.
+
+    It does so by querying the AWS EC2 API for the placement group by name.
+    :param placement_group_name: Name of the placement group
+    :return: True if placement group exists, False if placement group does not exist or unknown ina  region
+    """
+    try:
+        return AWSApi.instance().ec2.describe_placement_group(placement_group_name)
+    except AWSClientError as e:
+        print(f"Failed with error: {e}")
```

Comment:
I don't think we want to print the error here. Should be in charge of @handle_client_exception